tools/dumpkey/dumpkey.cpp

```diff
@@ -257,8 +257,7 @@ int main(int argc, const char **argv)
                 manager->reset();
                 while (manager->lookup(true) && count--)
                 {
-                    offset_t pos;
```

Comment:
There is code above for special treatment of filepos field that may not be needed any more.

In fact, can we get rid of the type_filepos and all associated code altogether?

Reply:
I hope we can.  I'll add to next PR